src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -47,24 +48,25 @@ public void storeMessage(Message message) {
   }
 
   /**
-   * Gets messages posted by a specific user.
+   * Loads in messages from the query into a list
    *
-   * @return a list of messages posted by the user, or empty list if user has never posted a
+<<<<<<< HEAD
```

Comment:
Looks like you have a merge conflict? Here are some steps for resolving them: https://sites.google.com/codeustudents.com/spring-2019/codeu-toolbox/reference-guides/using-git#h.p_Ru9ON8xbx9fC

Reply:
Oh, shoot! I used Atom to resolve the merge conflict, but I don't think it was properly added to the commit even though I saved and tested it on mvn. D:
Everything should be fine now, hopefully.